src/main/java/org/radarbase/appserver/event/state/MessageStateEventListener.java

```diff
@@ -55,7 +59,9 @@ public MessageStateEventListener(ObjectMapper objectMapper,
      *
      * @param event the event to respond to
      */
-    @EventListener(value = NotificationStateEventDto.class)
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
we have had transaction lock issues with postgres (aws rds) when using `Propagation.REQUIRES_NEW` previously, @mpgxvii can you elaborate the issues we faced previously.

Reply:
Indeed, using TransactionalEventListener + Async was essential, because it only opens a new transaction once the old one is committed. In earlier tests with other permutations, the transactions would overlap and block the HikariCP thread pool.

Reply:
Ah ok I see, sounds good. Yes we had to remove the `REQUIRES_NEW` propagation previously because it was blocking other transactions and causing incosistent states.

Reply:
@mpgxvii perhaps we can test this with our stage cluster?